docs/introduction-to-smart-contracts.rst

```diff
@@ -256,14 +257,17 @@ Ethereum this is roughly every 17 seconds.
 
 As part of the "order selection mechanism" (which is called "mining") it may happen that
 blocks are reverted from time to time, but only at the "tip" of the chain. The more
-blocks that are added on top, the less likely it is. So it might be that your transactions
+blocks are added on top of a particular block, the less likely this block will be reverted. So it might be that your transactions
 are reverted and even removed from the blockchain, but the longer you wait, the less
 likely it will be.
 
 .. note::
-    Transactions are not guaranteed to happen on the next block or any future specific block, since it is up to the miners to include transactions and not the submitter of the transaction. This applies to function calls and contract creation transactions alike.
+    Transactions are not guaranteed to be included in the next block or any specific future block,
+    since it is not up to the submitter of a transaction, but up to the miners to determine in which block the transaction is included.
+    This applies to function calls and contract creation transactions alike.
```

Comment:
Function calls?

Reply:
Sorry, I don't get it. Do you propose to capitalize the `f`?

Reply:
I did not write this, but I think it refers to transactions being sent to a contract as opposed to contract creation transactions.

Reply:
I think better to remove this sentence, explaining it properly may take multiple sentences and perhaps not worth the effort at that part of the documentation.

This could easily be misinterpreted as calls made on web3.js, which executes on the client's side.

Reply:
I agree. Deleted.